client/src/templates/Challenges/utils/frame.ts

```diff
@@ -69,7 +69,7 @@ const createHeader = (id = mainPreviewId) => `
       }
       if(element && element.nodeName === 'A'){
         e.preventDefault();
-        window.parent.window.alert('Normally this link would bring you to another website! It works!' \n + "(" + element.href + ")");
+        window.parent.window.alert('Normally this link would bring you to another website! It works!' + 'This is a link to: ' + '(' + element.href + ')');
```

Comment:
```suggestion
        window.parent.window.alert('Normally this link would bring you to another website! It works!' + ' This is a link to: ' + '(' + element.href + ')');
```

Spacing:

<img width="360" alt="image" src="https://user-images.githubusercontent.com/63889819/169583241-691d0c31-b71b-4234-832a-6becfb01057e.png">